templates/waggledance.json

```diff
@@ -70,6 +70,20 @@
 		"softLimit": 65536,
 		"hardLimit": 65536
 	  }
-	]
+	],
+        "systemControls": [
+          {
+                "namespace": "net.ipv4.tcp_keepalive_time", 
+                "value": "200"
```

Comment:
The best would be to inject variables here so anyone can override based on their infrastructure.

Reply:
will there be a separate PR for K8S - https://github.com/ExpediaGroup/apiary-federation/blob/master/k8s.tf ?

Reply:
You cannot do this easily in Kubernetes, it's not allowed to set these flags.
You get errors like:
```
PodSecurityPolicy: unable
--
to admit pod: [pod.spec.securityContext.sysctls[0]: Forbidden: unsafe
sysctl "net.ipv4.tcp_keepalive_time" is not allowed
pod.spec.securityContext.sysctls[1]: Forbidden: unsafe sysctl
"net.ipv4.tcp_keepalive_intvl" is not allowed
pod.spec.securityContext.sysctls[2]: Forbidden: unsafe sysctl
"net.ipv4.tcp_keepalive_probes" is not allowed]
```

So we need to sort this out separately